tests/monitoring/monitoring_service/test_handlers.py

```diff
@@ -548,6 +549,41 @@ def test_action_monitoring_triggered_event_handler_with_sufficient_balance_does_
     assert context.monitoring_service_contract.functions.monitor.called is True
 
 
+def test_action_monitoring_triggered_event_handler_with_insufficient_reward_amount_does_not_trigger_monitor_call(  # noqa
+        context: Context,
+):
+    """ Tests that `monitor` is called when the ActionMonitoringTriggeredEvent is triggered and
+    user has sufficient balance in user deposit contract
+
+    Also a test for https://github.com/raiden-network/raiden-services/issues/29 , as the MR
+    is sent after the channel has been closed.
+    """
+    context = setup_state_with_closed_channel(context)
+
+    context.db.upsert_monitor_request(get_signed_monitor_request(nonce=6, reward_amount=0))
+
+    trigger_event = ActionMonitoringTriggeredEvent(
+        token_network_address=DEFAULT_TOKEN_NETWORK_ADDRESS,
+        channel_identifier=DEFAULT_CHANNEL_IDENTIFIER,
+        non_closing_participant=DEFAULT_PARTICIPANT2,
+    )
+
+    channel = context.db.get_channel(
+        trigger_event.token_network_address,
+        trigger_event.channel_identifier,
+    )
+    assert channel
+    assert channel.closing_tx_hash is None
+
+    context.user_deposit_contract.functions.effectiveBalance(
```

Comment:
Is this necessary for the test?

Reply:
seems necessary, test fails without it, which is indeed strange. Error message
```
E       TypeError: '>=' not supported between instances of 'Mock' and 'int'

src/monitoring_service/handlers.py:397: TypeError
```